sphinx/io.py

```diff
@@ -93,7 +95,8 @@ class SphinxStandaloneReader(SphinxBaseReader):
                   Locale, CitationReferences, DefaultSubstitutions, MoveModuleTargets,
                   HandleCodeBlocks, AutoNumbering, AutoIndexUpgrader, SortIds,
                   RemoveTranslatableInline, FilterSystemMessages, RefOnlyBulletListTransform,
-                  UnreferencedFootnotesDetector, SphinxSmartQuotes, ManpageLink
+                  UnreferencedFootnotesDetector, SphinxSmartQuotes, ManpageLink,
+                  SphinxDomains, DoctreeReadEvent, Versioning,
```

Comment:
To maintain such list is ugly. It would be better to replace this by API.
To do that, we have to refactor i18n feature. It uses another list of transforms.